### PR TITLE
[azure/graphactivitylogs] fix destination.geo.region_name mapping

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
## Proposed commit message

This PR is to add `destination.geo.region_name` mapping into ecs.yml to fix error:
```
object mapping for [destination.geo.region_name] tried to parse field 
[region_name] as object, but found a concrete value\"}, dropping event!"
```

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).
- [ ] I have verified that any added dashboard complies with Kibana's [Dashboard good practices](https://docs.elastic.dev/ux-guidelines/data-viz/dashboard-good-practices) 

## Related issues

- Relates https://github.com/elastic/integrations/issues/10848

### CODE REVIEW

**Review by gizas — APPROVED** [review]

File: packages/azure/data_stream/graphactivitylogs/fields/ecs.yml [hunk 1]
```diff
@@ -2,3 +2,5 @@
   external: ecs
 - name: source.geo.location
   external: ecs
+- name: destination.geo.region_name
```
[the remark this answers is not in the document]
Comment (kaiyan-sheng) [reply packages/azure/data_stream/graphactivitylogs/fields/ecs.yml:5]: I think maybe in the future? Right now if we remove `destination.geo.region`, I would consider this as a breaking change to users who are already using this field. WDYT?

**Review by zmoog — APPROVED** [review]

**Review by muthu-mps — APPROVED** [review]
code owner approval.

**Review by ShourieG — APPROVED** [review]
LGTM

**Pull request merged**